src/createSafePrepareTransactionRequest.ts

```diff
@@ -77,7 +77,7 @@ export const SafeProxyFactoryAbi = [
  */
 export type CreateSafePrepareTransactionRequestParams<TChain extends Chain | undefined> = {
   publicClient: PublicClient<Transport, TChain>;
-  account: PrivateKeyAccount;
+  account: PrivateKeyAccount | `0x${string}`;
```

Comment:
let's just switch to an account address and drop `PrivateKeyAccount` (and update jsdoc accordingly)

```suggestion
  account: Address;
```

Reply:
good idea!